### PR TITLE
DATAUP-725: Make a builder for creating an ObjectInformation from a ref

### PR DESCRIPTION
**Pull request by MrCreosote**
... so all builds work the same way. Missed this one originally.

Also move all the ref tests into the OI unit tests where they should be.

### CODE REVIEW

**Comment by MrCreosote** [comment]
Closed and reopened to get travis to run

**Comment by MrCreosote** [comment]
Hmm, Travis doesn't seem to be picking up builds right now.

File: src/us/kbase/workspace/test/kbase/IdentifierUtilsTest.java [hunk 1]
```diff
@@ -778,7 +778,7 @@ public void sucessObjectSpecIncludedFalseStrict() throws Exception {
 		final ObjectIdentifier oi =  ret.get(0);
 		
 		checkObjectIdentifier(oi, "foo", null, opt("bar"), EL, EI, "foo/bar", "bar", false);
-		assertThat("incorrect hasRefpath()", oi.hasRefPath(), is(false));
+		assertThat("incorrect hasRefPath()", oi.hasRefPath(), is(false));
```
Comment (ialarmedalien) [inline src/us/kbase/workspace/test/kbase/IdentifierUtilsTest.java:781]: PHEW!

**Review by ialarmedalien — APPROVED** [review]
👍 

Glad to see that `getRefpath` fix!

**Comment by MrCreosote** [comment]
You can't say I'm not a man of my word when it comes to things that take a completely trivial amount of effort

**Comment by MrCreosote** [comment]
Ah ha, there's travis

**Pull request merged**